Resources/doc/getting_started/installation.rst

```diff
@@ -0,0 +1,173 @@
+* `SonataPropelAdminBundle`_ (integrates Propel);
+* `SonataDoctrinePhpcrAdminBundle`_ (integrates the Doctrine PHPCR ODM).
+
+You can download them in the same way as the SonataAdminBundle. For instance,
+to download the SonataDoctrineORMAdminBundle, execute the following command:
+
+.. code-block:: bash
+
+    $ composer require sonata-project/doctrine-orm-admin-bundle "2.3.*"
+
+.. tip::
+
+    Don't know which to choose? Most new users prefer SonataDoctrineORMAdmin,
+    to interact with traditional relational databases (MySQL, PostgreSQL, etc).
+
+Step 2: Enable the Bundle
+-------------------------
+
+Then, enable the bundle and the bundles is relies on by adding the following
+line in the `app/AppKernel.php` file of your project:
+
+.. code-block:: php
+
+    // app/AppKernel.php
+
+    // ...
+    class AppKernel extends Kernel
+    {
+        public function registerBundles()
+        {
+            $bundles = array(
+                // ...
+
+                // The admin requires some twig functions defined in the security
+                // bundle, like is_granted. Register this bundle if it wasn't the case
+                // already.
+                new Symfony\Bundle\SecurityBundle\SecurityBundle(),
+
+                // These are the other bundles the SonataAdminBundle relies on
+                new Sonata\CoreBundle\SonataCoreBundle(),
+                new Sonata\BlockBundle\SonataBlockBundle(),
+                new Knp\Bundle\MenuBundle\KnpMenuBundle(),
+
+                // And finally, the storage and SonataAdminBundle
+                new Sonata\DoctrineORMAdminBundle\SonataDoctrineORMAdminBundle(),
+                new Sonata\AdminBundle\SonataAdminBundle(),
+            );
+
+            // ...
+        }
+
+        // ...
+    }
+
+.. note::
+
+    If a bundle is already registered somewhere in your ``AppKernel.php``, you
+    should not register it again.
+
+.. note::
+
+    Since version 2.3, the bundle comes with jQuery and other front-end libs.
+    To update the versions (which isn't required), you can use `Bower`_. To
+    make sure you get the dependencies that match the version of
+    SonataAdminBundle you are using, you can make bower use the local bower
+    dependency file, like this:
+
+    .. code-block:: bash
+
+        $ bower install ./vendor/sonata-project/admin-bundle/bower.json
+
+Step 3: Configure the Installed Bundles
+---------------------------------------
+
+Now all needed bundles are downloaded and registered, you have to add some
+configuration. The admin interface is using SonataBlockBundle to put everything
+in blocks. You just have to tell the block bundle about the existence of the
+admin block:
+
+.. code-block:: yaml
+
+    # app/config/config.yml
+    sonata_block:
+        default_contexts: [cms]
+        blocks:
+            # enable the SonataAdminBundle block
+            sonata.admin.block.admin_list:
+                contexts:   [admin]
```

Comment:
For the installation, I think it's way out of scope. It might be usefull to explain more about it in another article though (but I'm not the guy to do that, I don't understand it either).